fuzzing/macros/macros.cmake

```
@@ -17,16 +17,32 @@ if(NOT "${BOLOS_SDK}/fuzzing" STREQUAL ${CMAKE_SOURCE_DIR})
 
   # Building from SDK
 else()
-  if("${TARGET_DEVICE}" STREQUAL "stax")
-    message("${Blue}Importing macros from ${CMAKE_SOURCE_DIR}/macros/macros-stax.txt")
-    file(STRINGS "${CMAKE_SOURCE_DIR}/macros/macros-stax.txt" MACRO_LIST)
-    list(APPEND MACRO_LIST "FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION=1")
-    target_compile_definitions(macros INTERFACE ${MACRO_LIST})
-  elseif("${TARGET_DEVICE}" STREQUAL "flex")
-    message("${Blue}Importing macros from ${CMAKE_SOURCE_DIR}/macros/macros-flex.txt")
-    file(STRINGS "${CMAKE_SOURCE_DIR}/macros/macros-flex.txt" MACRO_LIST)
+  if(${TARGET_DEVICE} STREQUAL "stax" OR ${TARGET_DEVICE} STREQUAL "flex")
+    # Run `make list-defines` and capture output
+    execute_process(
+      COMMAND sh -c "BOLOS_SDK=\${STAX_SDK} && TARGET=${TARGET_DEVICE} && CC=clang make list-defines"
+      WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/macros"
+      OUTPUT_VARIABLE MACRO_OUTPUT
+      OUTPUT_STRIP_TRAILING_WHITESPACE
+      ERROR_QUIET
+    )
```

Let's try to get as far as possible from platform specific commands.
Cmake has a few commands/functinos for such use cases.
Here we could try something like cmake env

```suggestion
execute_process(
  COMMAND ${CMAKE_COMMAND} -E env
    BOLOS_SDK=${STAX_SDK}
    TARGET=${TARGET_DEVICE}
    CC=clang
    make list-defines
  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/macros"
  OUTPUT_VARIABLE MACRO_OUTPUT
  OUTPUT_STRIP_TRAILING_WHITESPACE
  ERROR_QUIET
)
```